site/src/theme/components/MuiInputLabelThemeOptions.ts

```diff
@@ -0,0 +1,20 @@
+import { Components } from "@mui/material";
+import { defaultTheme } from "../util";
+
+export const MuiInputLabelThemeOptions: Components["MuiInputLabel"] = {
+  defaultProps: {
+    disableAnimation: true,
+    shrink: true,
+  },
+  styleOverrides: {
+    root: {
+      position: "unset",
+      left: "unset",
+      top: "unset",
+      transform: "unset",
+      fontSize: 15,
```

Comment:
I think it should be possible to set this based on our typography config. A simple approach I can think off is probably exporting the different font sizes from MuiTypography override file and using that here

Reply:
Good idea, if I'm understanding correctly we can just do `fontSize: theme.typography.bpSmallCopy.fontSize`. Have done this, and made similar changes to other files where I think it makes sense, here: https://github.com/blockprotocol/blockprotocol/pull/203/commits/33f414cba3942c30f6da2c909c1e3b55e98729a7